### PR TITLE
Update MeteorJS tooling and docker build steps

### LINKED ISSUE #12: Update MeteorJS tooling and docker build steps
Upped on patch version for MeteorJS.
Updated some build specifics in Dockerfile, mainly for working around
the hurdles caused by the old versions of MeteorJS and Node.js

### PR DESCRIPTION
**Pull request by yilanhung**
- Upped one patch version for MeteorJS.
- Updated some build specifics in Dockerfile, mainly for working around
  the hurdles caused by the old versions of MeteorJS and Node.js

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,30 +1,35 @@
-FROM node:5.0
-MAINTAINER Daniel Schaefer <daniel.schaefer@hpe.com>
+# This project is currently built against MeteorJS 1.4.2.7, which bundles
+# Node.js 4.4.7. The most recent and compatible Node.js image was picked to
+# provide the most compoatible build tooling for `meteor npm install'.
+FROM node:4.9.1-stretch as builder
 ENV SAPPORO_VERSION 0.1.0
 
-# Install old version of MeteorJS
```
Comment (JennieHo-HPE) [inline Dockerfile:5]: I'm curious that did they encounter the problem with the newer version?

File: Dockerfile [hunk 2]
```diff
@@ -1,30 +1,35 @@
-FROM node:5.0
-MAINTAINER Daniel Schaefer <daniel.schaefer@hpe.com>
+# This project is currently built against MeteorJS 1.4.2.7, which bundles
+# Node.js 4.4.7. The most recent and compatible Node.js image was picked to
+# provide the most compoatible build tooling for `meteor npm install'.
+FROM node:4.9.1-stretch as builder
```
Comment (JennieHo-HPE) [inline Dockerfile:4]: May I know why we choose the node:4.9.1-stretch instead of 5.0

Comment (yilanhung) [reply Dockerfile:4]: Read me comment above "FROM".

File: Dockerfile (hunk 1 above)
Comment (yilanhung) [reply Dockerfile:5]: What problem?

**Pull request merged**